src/Login.java

```diff
@@ -0,0 +1,32 @@
+public class Login {
+
+    private String username;
+    private String password;
+
+    public Login(String username, String password) {
+        this.username = username;
+        this.password = password;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public String toString() {
+        return "Username: " + this.username + " Password: " + this.password + "\n" +
```

Comment:
what is this, its not a xmas time yet! please change it to sth related to halloween and cats, 
thank you
Valentine

Reply:
done.